app/controllers/api/v2/applications_controller.rb

```diff
@@ -1,41 +1,5 @@
 module API
   module V2
-    class ApplicationsController < BaseController
-      include Pagination
-      include ::API::Concerns::FilterByUpdatedSince
-
-      def index
-        render json: to_json(paginate(applications_query.applications))
-      end
-
-      def show
-        render json: to_json(applications_query.application(ecf_id: application_params[:ecf_id]))
-      end
-
-      def accept = head(:method_not_allowed)
-      def reject = head(:method_not_allowed)
-
-    private
-
-      def applications_query
-        Applications::Query.new(
-          lead_provider: current_lead_provider,
-          cohort_start_years:,
-          updated_since:,
-        )
-      end
-
-      def cohort_start_years
-        application_params.dig(:filter, :cohort)
-      end
-
-      def application_params
-        params.permit(:ecf_id, filter: %i[cohort updated_since])
-      end
-
-      def to_json(obj)
-        ApplicationSerializer.render(obj, root: "data")
-      end
-    end
+    class ApplicationsController < V1::ApplicationsController; end
```

Comment:
I think we should avoid this kind of coupling in favour of duplication but happy to get a wider opinion

Reply:
Hi @ethax-ross, well api v1 & v2 are pretty much the same with minor diferences.. v3 can inherit most of v1 as well.. with the main diferences in the serializers & extra params. Could revert the changes and happy to get a wider opinion as well.

Reply:
I think I'd prefer keeping the duplication, as we agreed in the thread previously in the separation channel, even though they are similar we want to avoid this coupling especially when we deprecate versions. However I'm happy to see how this evolves as we go, and how we can tidy this up further as we build more endpoints with duplication initially in place. Let me know your thoughts :)

Reply:
Fair enough guys.. just have reverted in https://github.com/DFE-Digital/npq-registration/pull/1333/commits/bc7fbbe185096007c2c7e161b8aca2d67af2fcd3 👍 😄